subt/doc/virtual-howto.md

```diff
@@ -0,0 +1,61 @@
+# SubT Challenge - Virtual Track HowTo
+
+Here are instructions how to build, run locally and deploy to AWS "robotika" solution docker.
+It is expected that you already have docker installed and your PC has compatible NVidia GPU.
+For official instructions how to start with Docker please look at:
+  https://bitbucket.org/osrf/subt/wiki/tutorials/SystemSetupDockerInstall
+
+
+## Prepare OSGAR build environment
+The convenience scripts for build and run are merged into OSGAR:
+```
+git clone https://github.com/robotika/osgar.git .
+cd ~/osgar/subt/docker
+```
+
+## Download official docker images
+The currently available images are listed at:
+  https://hub.docker.com/r/osrf/subt-virtual-testbed/tags
+Note, that you will need cloudsim_sim_latest, cloudsim_bridge_latest and subt_solution_latest
+```
+docker pull osrf/subt-virtual-testbed:cloudsim_sim_latest
+docker pull osrf/subt-virtual-testbed:cloudsim_bridge_latest
+docker pull osrf/subt-virtual-testbed:subt_solution_latest
+```
+
+## Build and run locally simple unittest image
+```
+cd ~/osgar/subt/docker
+./build.bash unittest
+./run.bash unittest
+```
+
+It is necessary to run also simulation and bridge in other two terminals:
```

Comment:
Could we have a single script that runs all necessary commands? (this may be a topic for a separate change)

Reply:
There is "docker_compose" way, but then you lose access to recorded logfiles. It is possible but was not further used.